datafusion/common/src/hash_utils.rs

```diff
@@ -214,22 +214,19 @@ fn hash_struct_array(
     hashes_buffer: &mut [u64],
 ) -> Result<()> {
     let nulls = array.nulls();
-    let num_columns = array.num_columns();
+    let row_len = array.len();
 
-    // Skip null columns
```

Comment:
I forgot to correct the comment last time. Indices are all about row, not columns